packages/core/ios/Sources/DdLogs.h

```diff
@@ -0,0 +1,24 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+#import <Foundation/Foundation.h>
+@class DdLogsImplementation;
+
+#ifdef RCT_NEW_ARCH_ENABLED
+
+#import <DdSdkReactNative/DdSdkReactNative.h>
+@interface DdLogs: NSObject <NativeDdLogsSpec>
+
+#else
+
+#import <React/RCTBridgeModule.h>
+@interface DdLogs : NSObject <RCTBridgeModule>
+
+#endif
+
+@property (nonatomic, strong) DdLogsImplementation* ddLogsImplementation;
```

Comment:
I might be wrong, but I don't think you need to expose this in the `.h`, it can stay private to the class.

It's also the case in all declaration, `DdRum`, `DdTrace`, ...